quickwit/quickwit-proto/src/types.rs

```diff
@@ -171,6 +171,12 @@ impl Borrow<NodeIdRef> for NodeId {
     }
 }
 
+impl Default for NodeId {
```

Comment:
Let's remove `Default` on  `ShardTableEntry` instead.

Reply:
We need the default on NodeID because of the default trait used for `ShardTable`. Is that what you meant?

Reply:
Yes. Sorry.